en/core-api/cardless-credit.md

```diff
@@ -16,48 +15,51 @@ Basic integration process of Cardless Credit will be explained below.
+#### Endpoints
 
-#### Request Details
-Type | Value
---- | ---
-HTTP Method | `POST`
-API endpoint (Sandbox) | `https://api.sandbox.midtrans.com/v2/charge`
-API endpoint (Production) | `https://api.midtrans.com/v2/charge`
+| Environment | Method | URL                                        |
+| ----------- | ------ | ------------------------------------------ |
+| Sandbox     | POST   | https://api.sandbox.midtrans.com/v2/charge |
+| Production  | POST   | https://api.midtrans.com/v2/charge         |
 
-#### HTTP Headers
-```
-Accept: application/json
-Content-Type: application/json
-Authorization: Basic AUTH_STRING
-```
+#### Headers
 
-**AUTH_STRING**: Base64(`ServerKey + :`)
+| Header Name   | Description                                            | Required | Values                |
+| ------------- | ------------------------------------------------------ | -------- | --------------------- |
+| Accept        | The format of the data to be returned.                 | Required | application/json      |
+| Content-Type  | The format of the data to be posted.                   | Required | application/json      |
+| Authorization | The authentication method used to access the resource. | Required | Basic **AUTH_STRING** |
 
-?> HTTP request by using Basic Authentication method. The username is your Server Key while the password is empty. The authorization header value is represented by AUTH_STRING. AUTH_STRING is base-64 encoded string of your username & password separated by **:** (colon symbol).
+**AUTH_STRING**: Base64(`ServerKey + :`)<br>Midtrans API validates HTTP request by using Basic Authentication method. The username is your *Server Key* while the password is empty. The authorization header value is represented by AUTH_STRING. AUTH_STRING is base-64 encoded string of your username & password separated by a colon symbol (**:**). For more details, refer to [ API Authorization and Headers](https://docs.midtrans.com/en/technical-reference/api-header).
 
+?> ***Note***: *Server Key* is required to authenticate the request. For more details, refer to [HTTPS Header](https://api-docs.midtrans.com/#http-s-header).
 
+#### POST Body
 
-### 1. Send Transaction Data to API Charge
+| Element             | Description                                                  | Type   | Required |
+| ------------------- | ------------------------------------------------------------ | ------ | -------- |
+| payment_type        | Cardless Credit payment type.                                | String | Required |
+| transaction_details | The details of the transaction like the order_id and gross_amount. | -      | Required |
+| order_id            | The order_id of the transaction.                             | String | Required |
+| gross_amount        | The total amount of transaction.                             | Long   | Required |
```

Comment:
- Can you put this attributes explanation table not above the `Sample Request`, but below the `Sample Request`? looking at sample first is easier than looking at explanation first in this case. Just like the `Response Body` section.
- Can you put it within collapsible `<details>` element? we don't want to show this by default. Just like the `Response Body` section.
- Also it is not `POST Body` table, but more accurately `POST JSON Body attribute description`.